admin-manual/installation-setup/installation/install-ubuntu.rst

```diff
@@ -363,8 +363,21 @@ the configuration of your new server.
    only to ports 80 and 8000 for Archivematica usage.
 
 2. The Storage Service has its own set of users. Navigate to
-   **Administration > Users** and add at least one administrative user. After
-   you have created this user an API key will be generated that will connect
+   **Administration > Users** and add at least one administrative user:
+   ::
+
+      ::
+
+      sudo -u archivematica bash -c " \
+          set -a -e -x
+          source /etc/default/archivematica-storage-service || \
+              source /etc/sysconfig/archivematica-storage-service \
+                  || (echo 'Environment file not found'; exit 1)
+          cd /usr/lib/archivematica/storage-service
+          /usr/share/archivematica/virtualenvs/archivematica-storage-service/bin/python manage.py createsuperuser
+        ";
+
+   After you have created this user an API key will be generated that will connect
```

Comment:
I think that there is a better way to introduce literal blocks in reStructuredText:

```rst
2. The Storage Service has its own set of users. Navigate to
   **Administration > Users** and add at least one administrative user::

      sudo -u archivematica bash -c " \
          set -a -e -x
          source /etc/default/archivematica-storage-service || \
              source /etc/sysconfig/archivematica-storage-service \
                  || (echo 'Environment file not found'; exit 1)
          cd /usr/lib/archivematica/storage-service
          /usr/share/archivematica/virtualenvs/archivematica-storage-service/bin/python manage.py createsuperuser
        ";
```

Note the use of `::` only once. There is a good explanation in [the docs](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks):

> The handling of the :: marker is smart:
> If it occurs as a paragraph of its own, that paragraph is completely left out of the document.
> If it is preceded by whitespace, the marker is removed.
> If it is preceded by non-whitespace, the marker is replaced by a single colon.